RELEASE/scripts/autoscend/auto_cooking.ash

```diff
@@ -773,6 +814,10 @@ boolean loadConsumables(string _type, ConsumeAction[int] actions)
 	{
 		use(item_amount($item[van key]), $item[van key]);
 	}
+	if ((item_amount($item[Knob Goblin lunchbox]) > 0) && (pulls_remaining() != -1))
```

Comment:
pulls remaining -1 check is to disable it in casual or postronin.
But rather than disabling that, I would instead count the amount of the contents of the lunchbox. pseudocode: `if remaining liver < drink in lunchbox && remaining stomach < food in lunchbox`

Reply:
i don't want to open all of someone's lunchboxes in casual if they weren't acquired from a semi-rare, i'd rather open none because they're almost assuredly going to have better consumables anyway

Reply:
I am not suggesting we open all of them. Hence my suggested if check to limit how many we open. I would rather not open zero lunchboxes though. they probably have better but not certainly.

That since, since we now eat items one at a time, my suggestion was wrong. A better suggestion would be (one second, composing it)

Reply:
I'd still prefer just to not open any lunch boxes in casual, like how unremarkable duffel bag and van key's are already handled.